cmake/config_validation.cmake

```
@@ -29,12 +36,17 @@ if(CMAKE_HOST_WIN32)
     set(JAVA_EXE "$ENV{CUBE_PATH}\\STM32CubeMX.exe")
     set(CUBE_JAR "$ENV{CUBE_PATH}\\jre\\bin\\java.exe")
     set(JLINK_EXE JLink.exe)
+    set(OPENOCD_SCRIPTS_PATH "C:\\msys64\\mingw64\\share\\openocd\\scripts")
 else()
     set(JAVA_EXE $ENV{CUBE_PATH}/jre/bin/java)
     set(CUBE_JAR $ENV{CUBE_PATH}/STM32CubeMX)
     set(JLINK_EXE JLinkExe)
+    set(OPENOCD_SCRIPTS_PATH "/usr/share/openocd/scripts")
```

então, esses paths das configuracões do openocd são os padrões quando vc baixa pelo apt no linux ou pelo mingw64 no windows. Mas se vc tiver usado outro gerenciador de pacote ou tiver instalado manualmente, se pa não vai ser esse o caminho.

acho que da pra botar um

```suggestion
    if(NOT DEFINED ENV{OPENOCD_SCRIPTS_PATH})
        set(OPENOCD_SCRIPTS_PATH "C:\\msys64\\mingw64\\share\\openocd\\scripts")
    endif()
else()
    set(JAVA_EXE $ENV{CUBE_PATH}/jre/bin/java)
    set(CUBE_JAR $ENV{CUBE_PATH}/STM32CubeMX)
    set(JLINK_EXE JLinkExe)
    if(NOT DEFINED ENV{OPENOCD_SCRIPTS_PATH})
        set(OPENOCD_SCRIPTS_PATH "/usr/share/openocd/scripts")
    endif()
```

---

dai deixa no readme que precisa dar export nessa variaveis